Task6_GettingStartedwithAWS/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "AwsConfiguration": {
+    "AWSAccesskey": "AKIARKDSGBF6YGLNWV77",
+    "AWSSecretKey": "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"
+  },
+  "DatabaseConfiguration": {
```

Comment:
Why database credentials are here?

Reply:
removed as I implemented this earlier for testing.